app/controllers/api/external_api_keys_controller.rb

```diff
@@ -2,7 +2,20 @@ class Api::ExternalApiKeysController < Api::ApplicationController
   before_action :set_external_api_key, except: %i[index create]
 
   def index
-    @external_api_key = policy_scope(ExternalApiKey)
+    # Only authoris\zed to query own jurisdiction for review managers
+    if current_user.review_manager? && params[:jurisdiction_id].present? &&
+         current_user.jurisdiction_id != params[:jurisdiction_id]
+      raise Pundit::NotAuthorizedError
```

Comment:
You can delegate this check to the correct policy file
and call it like

authorize :external_api_key, :index?

Reply:
@bkmaibach I need the `params`, it's not available in the index? method in the policy